templates/taxi/manufacturer_list.html

```diff
@@ -7,7 +8,13 @@ <h1>
       Create
     </a>
   </h1>
-
+  <div>
+      <form action=" " method="get" class="form-inline">
```

Comment:
The `action` attribute in the form tag is currently set to a space. Consider setting it to an empty string or the current URL to ensure the form submits to the correct endpoint.